server/src/main/java/org/elasticsearch/index/IndexMode.java

```diff
@@ -229,6 +229,77 @@ public void validateSourceFieldMapper(SourceFieldMapper sourceFieldMapper) {
         public boolean isSyntheticSourceEnabled() {
             return true;
         }
+    },
+    LOGS("logs") { // TODO: stub waiting for PR #108896
```

Comment:
I added this while waiting for #108896 just to have the `LOGS` enum value defined.